svc_translation.go

```diff
@@ -30,11 +32,12 @@ type Translation struct {
 	ModifiedAtTs    int64  `json:"modified_at_timestamp"`
 	ModifiedBy      int64  `json:"modified_by"`
 	ModifiedByEmail string `json:"modified_by_email"`
-	IsFuzzy         bool   `json:"is_fuzzy"`
```

Comment:
Breaking changes

Reply:
Yes, because the current version is invalid and won't work with Lokalise Api